examples/object_detection_3d/object_detection_3d/evaluator.py

```diff
@@ -77,11 +86,53 @@ def name(self):
         return name
 
 
+@dataclass(frozen=True)
+class UnmatchedLabeledBoundingBox3D(LabeledBoundingBox3D):
+    max_overlap: float
```

Comment:
Added `max_overlap` for FP and FN because I wasted a day thinking metrics implementation may be wrong because some boxes look to have high overlap visually. Turned out visual is deceiving and overlap is indeed below threshold.